### PR TITLE
add 171. Excel Sheet Column Number

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m
Please check my code.

### CODE REVIEW

File: 171_excel-sheet-column-number.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+
+# @lc code=start
+class Solution:
+    def titleToNumber(self, columnTitle: str) -> int:
+        """
+        time: O(len(columnTitle))
+        space: O(len(columnTitle))
+        """
+        ref = {"A": 1, "B": 2, "C": 3, "D": 4, "E": 5, "F": 6, "G": 7, "H": 8, "I": 9, "J": 10, "K": 11, "L": 12, "M": 13, "N": 14, "O": 15, "P": 16, "Q": 17, "R": 18, "S": 19, "T": 20, "U": 21, "V": 22, "W": 23, "X": 24, "Y": 25, "Z": 26, }
```
Comment (ryuji0123) [inline 171_excel-sheet-column-number.py:16]: use chr and ord functions

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I used chr functions.  Please check my code.

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
avoid magic numbers

**Comment by kobakobashu** [comment]
I avoided magic number: 26. Please check my code.

File: 171_excel-sheet-column-number.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# @lc code=start
+class Solution:
+    def titleToNumber(self, columnTitle: str) -> int:
+        """
+        time: O(len(columnTitle))
+        space: O(len(columnTitle))
+        """
+        ref = {}
+        for i in range(1, 27):
```
Comment (tkkawa) [inline 171_excel-sheet-column-number.py:17]: -> for i in range(1, num_alp+1):

**Review by tkkawa — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I modified my code. Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request closed without merging**